cognite/neat/_graph/extractors/_classic_cdf/_base.py

```diff
@@ -182,9 +223,27 @@ def _get_rdf_type(self, item: T_CogniteResource) -> str:
 
     def _as_object(self, raw: Any, key: str) -> Literal | URIRef:
```

Comment:
```suggestion
    def _as_object(self, raw: Any, key: str) -> Literal | URIRef:
        """Return properly formatted object part of s-p-o triple"""
```

Old guy needs to understand it is object of triple not python object :)